src/studio/src/designer/frontend/shared/utils/urlHelper.ts

```diff
@@ -1,22 +1,23 @@
 import getNamesFromLocation from './getNamesFromLocation';
+import conditionalPath from "app-shared/utils/conditionalPath";
 
 const cdn = 'https://altinncdn.no';
-export const orgsListUrl = `${cdn}/orgs/altinn-orgs.json`;
```

Comment:
This seems to be in use, I don't think it can be deleted.

Reply:
Could not find usage in the code, intellij marked it as unused. Why aren't tests failing?

Reply:
The value is still defined in a different file